webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -48,9 +49,29 @@ export const newWindowOnClick = url => event => {
  */
 export const clearSpaces = string => string.trim().replace(/\s\s+/, ' ');
 
+/**
+ * Use I18n to translate an object of strings
+ * @param {Object.<string, string>} obj - the object to translate
+ * @returns {Object.<string, string>} a translated object
+ */
+export const translateObject = obj =>
+  Object.assign(
+    {},
+    ...Object.entries(obj).map(([k, v]) => ({ [k]: translate(v) }))
+  );
+
+/**
+ * Use I18n to translate an array of strings
+ * @param {Array.<string>} arr - the array to translate
+ * @returns {Array.<string>} a translated array
+ */
+export const translateArray = arr => arr.map(str => translate(str));
```

Comment:
I think those functions should live in the I18n file:
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/I18n.js